tannerb9/PRIX-client
# Add Ingredient Form

# Description
Created an 'Add Ingredient' button on IngredientList.js that opens a form(modal) and allows the user to add an ingredient.
BUG: after form submits and modal closes, IngredientList doesn't update

## Type of change

# Testing Instructions
- Log in and go the 'Ingredients' tab.
- Click the 'Add Ingredient' button. 
- Fill out the form and hit submit. 
- It should close the modal and return you to the 'Ingredients' tab, but you'll have to refresh the page to see that the ingredient has been added to the bottom of the list.

# Checklist:
